archimedes/meta_obj.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2014-2018 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Authors:
+#   Valerio Cosentino <valcos@bitergia.com>
+#
+
+import json
+
+
+class MetaObj:
+    """This class models the metadata information of Kibana objects which includes the ID, title, type and version
+    of the object plus when the object was updated if not null.
+
+    :param _id: object id
+    :param _title: object title
+    :param _type: object type
+    :param _version: object version
+    :param _updated_at: time when the object was updated, it can be null
+    """
+    def __init__(self, _id, _title, _type, _version, _updated_at=None):
```

Comment:
Why are you using variables starting with an underscore? In the case it's because name collision problems, it's not a problem because you are assigning these variables to `self` attributes.